packages/perseus/src/__testdata__/server-item-renderer.testdata.ts

```diff
@@ -40,7 +41,7 @@ export const itemWithNumericInput: PerseusItem = {
                     labelText: "What's the answer?",
                     size: "normal",
                 },
-            } as NumericInputWidget,
+            } satisfies NumericInputWidget,
```

Comment:
`satisfies` 

> The new satisfies operator lets us validate that the type of an expression matches some type, without changing the resulting type of that expression. 

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html#the-satisfies-operator

Compare to `as` which does some type checking but is not as robust and can end up hiding test data that doesn't actually fully match the `as` type.